app/src/main/res/values/strings.xml

```diff
@@ -289,7 +289,7 @@
     <string name="running_apps">Running Apps</string>
     <string name="no_running_apps">No running apps</string>
     <string name="kill_process">Kill</string>
-    <string name="disable_background_run">Prevent Running in the Background</string>
+    <string name="disable_background_run">Keep App in Foreground</string>
```

Comment:
This translation is even more misleading. **Disable Run in Background** option disables the **RUN_IN_BACKGROUND** app op which prevents an app from running in background. **Keep App in Foreground** doesn't go with this. Also, **app** should be plural (better to omit it entirely).